snippets/network-multiple-subnet-control-plane-nodes.adoc

```diff
@@ -0,0 +1,18 @@
+// Text snippet included in the following assemblies:
+//
+//
+//
+// Text snippet included in the following modules:
+//
+// * modules/ipi-install-establishing-communication-between-subnets.adoc
+// * modules/ipi-install-configuring-host-network-interfaces-for-subnets.adoc
+
+:_mod-docs-content-type: SNIPPET
+[IMPORTANT]
+====
+You can run control plane nodes in the same subnet or multiple subnets. If you run a multiple subnet environment, you can reduce the risk of your {product-title} cluster from failing due to a hardware failure or a network outage.
+
+For a multiple subnet environment, create a manifest configuration file and specify the external load balancer address in the `ingressVIPs` and `apiVIPs` parameters of the file. Configure an external load balancer instead of the default load balancer by specifying `UserManaged` in the `loadBalancer.type` parameter of the `install-config.yaml` file. For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".
```

Comment:
```suggestion
For a multiple subnet environment, We need to specify the external load balancer.
1. Configure an external load balancer address in the `ingressVIPs` and `apiVIPs` parameters of the install-config.yaml. 
2. Configure an external load balancer instead of the default load balancer by specifying `UserManaged` in the `loadBalancer.type` parameter of the `install-config.yaml` file. 
For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".
```

Reply:
in `Services for a user-managed load balancer` page, the haproxy example as blow is not suitable, the masters ip (192.168.1.101/192.168.1.102/192.168.1.103) are in same subnet.  however this feature need multi-subnet.  Can we create one another page to describe this ELB and multi-subnet feature?
```
#...
listen my-cluster-api-6443
    bind 192.168.1.100:6443
    mode tcp
    balance roundrobin
  option httpchk
  http-check connect
  http-check send meth GET uri /readyz
  http-check expect status 200
    server my-cluster-master-2 192.168.1.101:6443 check inter 10s rise 2 fall 2
    server my-cluster-master-0 192.168.1.102:6443 check inter 10s rise 2 fall 2
    server my-cluster-master-1 192.168.1.103:6443 check inter 10s rise 2 fall 2
```